workitem/workitem_repository_blackbox_test.go

```diff
@@ -296,8 +296,33 @@ func (s *workItemRepoBlackBoxTest) TestCodebaseAttributes() {
 	assert.Equal(s.T(), title, wi.Fields[workitem.SystemTitle].(string))
 	require.NotNil(s.T(), wi.Fields[workitem.SystemCodebase])
 	cb := wi.Fields[workitem.SystemCodebase].(codebase.CodebaseContent)
-	assert.Equal(s.T(), repo, cb.Repository)
+	expectedRepo := repo + ".git"
+	assert.Equal(s.T(), expectedRepo, cb.Repository)
 	assert.Equal(s.T(), branch, cb.Branch)
 	assert.Equal(s.T(), file, cb.FileName)
 	assert.Equal(s.T(), line, cb.LineNumber)
 }
+
+func (s *workItemRepoBlackBoxTest) TestCodebaseInvalidRepo() {
+	// given
+	title := "solution on global warming"
+	branch := "earth-recycle-101"
```

Comment:
yes that list is very very helpful. Thank you.
I will have it in tests depending on the response to https://github.com/almighty/almighty-core/pull/1241#discussion_r113361551